themes/default/content/case-studies/atlassian.md

```diff
@@ -1,11 +1,10 @@
 ---
-title: Atlassian
+title_tag: Bitbucket Cloud | Case Studies
+title: "Bitbucket: Increasing Developer Productivity"
```

Comment:
The case study itself talks about Bitbucket, so we felt it would be more accurate. That being said, Atlassian is the bigger name, so perfectly fine with having Atlassian remain the focus!

Reply:
lets go with Atlassian Bitbucket